src/MountedTraversal.js

```diff
@@ -47,6 +47,9 @@ export function instEqual(a, b, lenComp) {
 
 export function instHasClassName(inst, className) {
   const node = findDOMNode(inst);
+  if (!node) { // inst renders null
```

Comment:
But can a react component return anything other than `null` or a react element? Besides, what we're talking about here is the output of `findDOMNode`, can it be anything other than a dom element or `null`, I dont know...